scripts/shared/build_image.sh

```diff
@@ -36,7 +36,7 @@ latest_image=${repo}/${image}:latest
 cache_flag=''
 if [[ "$cache" = true ]]; then
     cache_flag="--cache-from ${latest_image}"
-    docker pull ${latest_image}
+    docker pull ${latest_image} || :
```

Comment:
Hah yes when we introduce a new image, this would have failed ;-).

Reply:
hehe

Reply:
I meant thumbs up... ':)